File: CPP/SessionData/Day-8/Address.cpp
#include<iostream>
#include<string>
using namespace std;

class Address 
{
	//DM
	private:
	string line1;
	string line2;
	string pin;

	//MF
	public:
	//constructor to intialize object
	//No-arg ctor or default ctor
	Address()
	{
	cout<<"\n Address::Address()"<<endl;
	this->line1 = "";
	this->line2 = "";
	this-> pin= "";
	}

	Address(string line1, string line2, string pin)
	{
	cout<<"\n Address::Address(string, string, string)"<<endl;
	this->line1 = line1;
	this->line2 = line2;
	this->pin = pin;
	}
	
	void Accept()
	{
		cout<<"\n please enter adresss data"<<endl;
	cin>>line1;
	cin>>line2;
	cin>>pin;
	}
	void Display() const
	{
		cout<<"\n Adresss data"<<endl;
	//Printing data
	cout<<"\n"<<line1<<", "<<line2<<", "<< pin ;
	}
};

File: OOPWJAVA/SessionData/Day8/EmployeeExampleV3/src/com/cdac/acts/Address.java
package com.cdac.acts;

public class Address {
	
	private String line1;
	private String line2;
	private String landmark;
	private String pin;
	
	public Address() {
	}

	public Address(String line1, String line2, String landmark, String pin) {
		this.line1 = line1;
		this.line2 = line2;
		this.landmark = landmark;
		this.pin = pin;
	}

	@Override
	public String toString() {
		return "Address [line1=" + line1 
				+ ", line2=" + line2 
				+ ", landmark=" + landmark 
				+ ", pin=" + pin + "]";
	}

}
